File: Strings/pagina03/2062/2062.java
import java.util.Scanner;
import java.io.IOException;

public class Main{
    public static void main(String[] args) throws IOException {
        Scanner in = new Scanner(system.in);

        int n = in.nextInt();
        in.nextLine();
        String[] words = in.nextLine().split(' ');
        String response = "";

        for(int i = 0; i < n; i++){
            String word = words[i];

            if(!response.isEmpty()){ response += ' '; }

            if(word.length() == 3){
                if(word.startsWith("OB")){
                    response += 'OBI';
                    continue;
                }
                if(word.startsWith("UR")){
                    response += 'URI';
                    continue;
                }
            }
            response += word;
        }

        System.oout.println(response);
        in.close();
    
    }
}

File: Strings/__2000__/2062/2062.py
n_words = int(input())
words = input().split()

correct_words = []
for word in words:
    if len(word) == 3:
        if word[:2] == 'OB':
            correct_words.append("OBI")
        elif word[:2] == 'UR':
            correct_words.append("URI")
        else:
            correct_words.append(word)
    else:
        correct_words.append(word)

print(" ".join(correct_words).strip())